build.gradle

```diff
@@ -160,6 +161,9 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-client'
```

Comment:
FYI if we are upgrading to Gradle 6 then the recommended configuration is a bit different (but can change later):
See: https://stackoverflow.com/questions/44493378/whats-the-difference-between-implementation-and-compile-in-gradle
```compile with implementation (if you don't need transitivity) or api (if you need transitivity)
testCompile with testImplementation
debugCompile with debugImplementation
androidTestCompile with androidTestImplementation
compileOnly is still valid. It was added in 3.0 to replace provided and not compile. (provided introduced when Gradle didn't have a configuration name for that use-case and named it after Maven's provided scope.)```

Reply:
tbh, I just changed Gradle version and check if the project compile and test are running. Let me dig into details

Reply:
Don't need to do it now ... for future reference